src/Acuminator/Acuminator.Vsix/packages.config

```diff
@@ -30,6 +31,7 @@
   <package id="Microsoft.VisualStudio.TextManager.Interop" version="7.10.6070" targetFramework="net461" />
   <package id="Microsoft.VisualStudio.TextManager.Interop.8.0" version="8.0.50727" targetFramework="net461" />
   <package id="Microsoft.VisualStudio.Threading" version="14.1.111" targetFramework="net461" />
+  <package id="Microsoft.VisualStudio.Threading.Analyzers" version="15.8.122" targetFramework="net461" />
```

Comment:
Is it compatible with VS2015?

Reply:
I haven't checked yet, I'll try it now. It is actually from ExtensibilityTolls extension I installed.

Reply:
As far as I can see, it is compatible with VS 2015. This package contains analyzers for wrok with VS SDK, some are very useful - thanks to them I added some unobvious checks to code